Makefile

```diff
@@ -48,6 +73,15 @@ ${BOARD}.img: boot/boot-${BOARD}.scr $(ROOTFS)
 	rm -f $@
 	tar -Szcf $@ $<
 
+## Convenience, avoiding the need to repeat yourself about which board
+img: $(BOARD).img
+tar: $(BOARD).tar
+tgz: $(BOARD).tar.gz
+
+dd: $(BOARD).img
+	sudo dd if=$(BOARD).img of=/dev/mmcblk0 bs=4096
```

Comment:
Since the command is potentially destructive and run with sudo I think we should at least warn the user and verify the device name (it could be something else than mmcblk0)